src/frontend/utils/dato.ts

```diff
@@ -15,20 +15,61 @@ export interface IPeriode {
 
 export const dagensDato = startOfToday();
 
-interface FormatterProps {
+export const tidenesMorgen = new Date(1000, 1, 1);
+
+export const tidenesEnde = new Date(3000, 1, 1);
+
+interface FormatterDateProps {
     dato?: Date;
     datoformat: Datoformat;
     defaultString?: string;
 }
-export const formatterDate = ({ dato, datoformat, defaultString = '' }: FormatterProps): string => {
+export const formatterDate = ({
```

Comment:
Burde denne hete `formatDate` eller `formaterDato`?

Reply:
Jepp! Tror jeg går for `formaterDate` så man vet at man jobber på et Date-objekt